observability/network_observability/network-observability-operator-release-notes.adoc

```diff
@@ -13,6 +13,31 @@ These release notes track the development of the Network Observability Operator
+* When the eBPF agent `PacketDrop` feature was enabled, and sampling was configured to a value greater than `1`, reported dropped bytes and dropped packets ignored the sampling configuration. While this was done on purpose, so as not to miss any drops, a side effect was that the reported proportion of drops versus non-drops became biased. For example, at a very high sampling rate, such as `1:1000`, it was likely that almost all the traffic appears to be dropped when observed from the console plugin. With this fix, the sampling configuration is honored with dropped bytes and packets. (link:https://issues.redhat.com/browse/NETOBSERV-1676[*NETOBSERV-1676*])
+* Previously, the SR-IOV secondary interface was not detected if the interface was created first and then the eBPF agent was deployed. It was only detected if the agent was deployed first and then the SR-IOV interface was created. Now, the SR-IOV secondary interface is detected no matter the sequence of the deployments. (link:https://issues.redhat.com/browse/NETOBSERV-1697[*NETOBSERV-1697*])
+* Previously, when Loki was disabled, the *Topology* view in the OpenShift web console displayed the *Cluster* and *Zone* aggregation options in the slider beside the network topology diagram, even when the related features are not enabled. This has been fixed, the slider now only display options according to the enabled features. (link:https://issues.redhat.com/browse/NETOBSERV-1705[*NETOBSERV-1705*])
+* Previously, when Loki was disabled, and the OpenShift web console was first loading, an error would occur: `Request failed with status code 400 Loki is disabled`. With this fix, the errors no longer occur. (link:https://issues.redhat.com/browse/NETOBSERV-1706[*NETOBSERV-1706*])
+* Previously, in the *Topology* view of the OpenShift web console, when clicking on the *Step into* icon next to any graph node, the filters were not applied as required in order to set the focus to the selected graph node, resulting in showing a wide view of the *Topology*. Now the filters are correctly set, effectively narrowing down the *Topology*. As part of this change, clicking the *Step into* icon on a *Node* now brings you to the *Resource* scope instead of the *Namespaces* scope. (link:https://issues.redhat.com/browse/NETOBSERV-1720[*NETOBSERV-1720*])
+* Previously, when Loki was disabled, in the *Topology* view of the OpenShift web console with the *Scope* set to *Owner*, clicking on the *Step into* icon next to any graph node would bring the *Scope* to *Resource*, which isn't available without Loki, so an error message was shown. Now the *Step into* icon is hidden in the *Owner* scope when Loki is disabled, so this scenario no longer occurs.(link:https://issues.redhat.com/browse/NETOBSERV-1721[*NETOBSERV-1721*])
```

Comment:
I guess this contraction should be removed per [SSG](https://redhat-documentation.github.io/supplementary-style-guide/#contractions):
```suggestion
* Previously, when Loki was disabled, in the *Topology* view of the OpenShift web console with the *Scope* set to *Owner*, clicking on the *Step into* icon next to any graph node would bring the *Scope* to *Resource*, which is not available without Loki, so an error message was shown. Now the *Step into* icon is hidden in the *Owner* scope when Loki is disabled, so this scenario no longer occurs.(link:https://issues.redhat.com/browse/NETOBSERV-1721[*NETOBSERV-1721*])
```